### ISSUE TITLE
Lintly Bot

### ISSUE THREAD

**Issue by uostim** [LintlyCI/Lintly-Issues#1]
Should the lintly bot automatically accept "collaborator" invitations and then automatically comment on commits?

How long does it take to accept and invitation?

Many thanks,
-uostim

**Comment by grantmcconnaughey** [comment]
Hey @uostim! @lintly-bot doesn't accept collaborator invitations automatically yet. I have to do it manually for now.

lintly-bot will only comment on pull requests. If it has been invited to be a collaborator then lintly-bot will comment on individual lines of a PR to show exactly where the issues are. Otherwise it will make only one comment on the PR as a whole.

I received the invitation for lintly-bot to join your repo but it appears that the invitation is no longer valid. If you re-invite lintly-bot then I'd be happy to accept.

I hope that clears things up!

Grant

**Comment by uostim** [comment]
Hi Grant,

Thanks for the reply!

The repo I invited @lintly-bot to just has me committing so far, so 
there wouldn't be any pull requests for it to comment on, but I'll 
certainly keep it in mind for future projects.

I've used your web front end to lint that same repository this morning, 
my very first time linting, and I have to say I was shocked at the 
amount of basic coding style errors I'd left.

Thanks for creating such a useful tool! :)

-Tim (@uostim)
